lib/rihanna.ex

```diff
@@ -82,7 +83,13 @@ defmodule Rihanna do
   Rihanna.enqueue(MyApp.MyJob, [arg1, arg2])
   ```
   """
-  @spec enqueue(module, any) :: {:ok, Rihanna.Job.t()}
+
+  @spec enqueue({module(), any()}, map()) :: {:ok, Rihanna.Job.t()}
+  def enqueue(term = {mod, _arg}, opts) when is_atom(mod) do
```

Comment:
This is surprising. Shouldn't it be `enqueue(mod, arg, opts)`?

Reply:
This is related to the comment from the first commit
https://github.com/samphilipd/rihanna/pull/82#discussion_r323043026
I think it can be changed if you do not want it like this.

Reply:
Looks fine, can you update docs/README as necessary